AirSkyBoat/AirSkyBoat
# Additional effect duration logic update + Lilith's Rod proc

**_I affirm:_**

## What does this pull request do?

* Add a proc to Lilith's Rod based on the wiki (up to 25 mp and 20-30% proc rate).
* Updates additional effect resistance logic to remove macc/meva bonuses, dStat and skill checks and rely on just resistance and dLvl. The intent is to make Additional Effects of WS last longer like they did on retail.

## Steps to test these changes

* Equip Lilith's Rod and smack things.
* Add logging to a WS LUA to see what the duration is. Smack a bunch of things with it until satisfied.
